cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -765,32 +765,88 @@
   </property>
 
   <property>
-    <name>kafka.log.dir</name>
+    <name>kafka.server.log.dir</name>
     <value>/tmp/kafka-logs</value>
     <description>
       CDAP Kafka service log storage directory
     </description>
   </property>
 
   <property>
-    <name>kafka.num.partitions</name>
+    <name>kafka.server.log.flush.interval.messages</name>
+    <value>10000</value>
+    <description>
+      The interval length at which we will force an fsync of data written to the log.
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.log.retention.hours</name>
+    <value>24</value>
+    <description>
+      The number of hours to keep a log file before deleting it (in hours)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.log.segment.bytes</name>
+    <value>536870912</value>
+    <description>
+      The maximum size of a single log file
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.num.partitions</name>
     <value>10</value>
     <description>
       Default number of partitions for a topic
     </description>
   </property>
 
   <property>
-    <name>kafka.seed.brokers</name>
+    <name>kafka.server.seed.brokers</name>
     <value>127.0.0.1:9092</value>
     <description>
-      Comma-separated list of CDAP Kafka service brokers; for distributed CDAP, 
+      Comma-separated list of CDAP Kafka service brokers; for distributed CDAP,
       replace with list of FQDN:port brokers
     </description>
   </property>
 
   <property>
-    <name>kafka.zookeeper.namespace</name>
+    <name>kafka.server.socket.receive.buffer.bytes</name>
+    <value>1048576</value>
+    <description>
+      The maximum number of bytes in a socket request
```

Comment:
I think these are out-of-order; from https://kafka.apache.org/08/configuration.html; should it be:
```
The size in bytes of the receive buffer the server prefers for socket connections